org.osgi.test.junit5/src/test/java/org/osgi/test/junit5/context/BundleContextExtension_InstallBundleSanityCheckingTest.java

```diff
@@ -6,8 +6,8 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.api.extension.ExtensionConfigurationException;
 import org.junit.jupiter.api.extension.ParameterResolutionException;
-import org.osgi.test.common.annotation.InjectInstallBundle;
-import org.osgi.test.common.install.InstallBundle;
+import org.osgi.test.common.annotation.InjectBundleInstaller;
+import org.osgi.test.common.install.BundleInstaller;
 
 public class BundleContextExtension_InstallBundleSanityCheckingTest {
```

Comment:
-> `BundleContextExtension_BundleInstallerSanityCheckingTest`

Reply:
renamed

Reply:
I think that you missed this one?

Reply:
yes. i renamed an other (wrong one instead)